glib/src/types.rs

```diff
@@ -246,6 +246,16 @@ impl StaticType for Type {
     }
 }
 
+pub trait StaticTypeExt {
```

Comment:
Please add it to the prelude and also add some docs :)

Reply:
Also add a doc alias :-)

Reply:
> Also add a doc alias :-)

I was thinking about that, but does it make sense considering I am not really binding `g_type_ensure`?

Reply:
It does the same, so yes

Reply:
Should be fine now.